modules/migration-state-migration-cli.adoc

```diff
@@ -3,12 +3,13 @@
 // * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
 // * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
 
+:_content-type: PROCEDURE
 [id="migration-state-migration-cli_{context}"]
 = State migration
 
-You can perform repeatable, state-only migrations by using {mtc-full} ({mtc-short}) to migrate persistent volume claims (PVCs) that constitute an application's state. You migrate specified PVCs by excluding other PVCs from the migration plan. Persistent volume (PV) data is copied to the target cluster. The PV references are not moved. The application pods continue to run on the source cluster. You can map the PVCs to ensure that the source and target PVCs are synchronized.
+You can perform repeatable, state-only migrations by using {mtc-full} ({mtc-short}) to migrate persistent volume claims (PVCs) that constitute an application's state. You migrate specified PVCs by excluding other PVCs from the migration plan. You can map the PVCs to ensure that the source and the target PVCs are synchronized. Persistent volume (PV) data is copied to the target cluster. The PV references are not moved, and the application pods continue to run on the source cluster.
 
-You can perform a one-time migration of Kubernetes objects that constitute an application's state.
+State migration is specifically designed to be used in conjuction with external CD mechanisms, such as OpenShift Gitops. You can migrate application manifests using GitOps, while migrating the state using {mtc-short}.
```

Comment:
We try to stay away from the term "designed to" (https://www.ibm.com/docs/en/ibm-style?topic=word-usage#t). Maybe try "can"? or another variation. We also try to stay away from the term "by using". Let's try, for example: 

State migration can be used in conjunction with external CD mechanisms, such as OpenShift Gitops. You can migrate application manifests by using GitOps, while migrating the state by using {mtc-short}.

Reply:
@stevsmit 
'Can' does not really convey the idea. How about 'intended to be used'?

I am confused about 'by using'. There is no 'by using' in my sentence, but you introduced it in your suggestion.

Reply:
Apologies @sbeskin-redhat . We try to stay away from the term "using", and suggest "by using" in its place. See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u for example. It's just a small nit. Sorry for the confusion.